marketplaceItems/voiceScopeApp/appResources/appData/voiceScope_app.js

```diff
@@ -51,6 +51,22 @@
                     value: enabled
                 });
                 break;
+            case "TOGGLE_DRAW":
```

Comment:
where were the Toggle Draw checkbox and Height Offset Slider features requested, documented, or scoped? These are additional features, not bugfixes

Reply:
Captured on the market place trello card, they were some of the immediate feedback I received from users and Philip in face to face conversations.